server/controller.py

```diff
@@ -825,6 +825,14 @@ def block_get_header(self, height):
         number = self.non_negative_integer(number)
         return await self.daemon_request('estimatefee', [number])
 
+    async def mempool_get_fees(self):
```

Comment:
This doesn't need to be async as it isn't (in the current implementation at least; perhaps it should be if it potentially involved a lazy histogram recompute).  The JSON infrastructure in ElectrumX does not require callbacks to be async - see server.features and block.get_chunk.